ci-operator/config/openshift/release/openshift-release-master__nightly-4.19.yaml

```diff
@@ -343,6 +343,20 @@ tests:
       enable:
       - observers-resource-watch
     workflow: baremetalds-e2e-ovn-ipv6
+- as: e2e-metal-ipi-serial-ovn-ipv6-techpreview
+  cluster: build05
+  cron: '@daily'
```

Comment:
Given metal's limited resources, I'd suggest using a cronline directly instead to run these at a non- busy time 
<img width="1082" alt="image" src="https://github.com/user-attachments/assets/130cbdf1-8b47-4936-9dc4-edaa1536c47e" />

Reply:
The rehearsals are also close to the timeout limit, do you want to add `shard_count: 2` to split it? Should be OK as long as they're being run at non-busy times